content/docs/components/serving/seldon.md

```diff
@@ -6,22 +6,20 @@ weight = 40
 
 {{% stable-status %}}
 
-## Serve a model using Seldon
-
-Seldon comes installed with Kubeflow. Full documentation for running Seldon inference is provided within the [Seldon documentation site](https://docs.seldon.io/projects/seldon-core/en/latest/).
+Seldon comes installed with Kubeflow. The [Seldon documentation site](https://docs.seldon.io/projects/seldon-core/en/latest/) provides full documentation for running Seldon inference.
 
 If you have a saved model in a PersistentVolume (PV), Google Cloud Storage bucket or Amazon S3 Storage you can use one of the [prepackaged model servers provided by Seldon](https://docs.seldon.io/projects/seldon-core/en/latest/servers/overview.html).
 
 Seldon also provides [language specific model wrappers](https://docs.seldon.io/projects/seldon-core/en/latest/wrappers/README.html) to wrap your inference code for it to run in Seldon.
 
-### Kubeflow Specifics
+## Kubeflow specifics
 
 You need to ensure the namespace where your models will be served has:
 
- 1. A Istio gateway named kubeflow-gateway
- 2. The namespace is labeled with `serving.kubeflow.org/inferenceservice=enabled`
+* An Istio gateway named kubeflow-gateway
+* A label set as `serving.kubeflow.org/inferenceservice=enabled`
 
-Examples:
+Example:
```

Comment:
How about rewording this and the next line, to avoid the double set of phrases ending in colons. Something like this:

```
The following example applies the label `my-namespace` to the namespace for serving:
```

Reply:
Good.